cabot/metricsapp/migrations/0008_auto__add_field_metricsstatuscheckbase_consecutive_failures.py

```diff
@@ -8,26 +8,15 @@
 class Migration(SchemaMigration):
```

Comment:
Diff is messed up.  The dummyapp was deleted (along with all migrations).  I also added a new metricsapp migration 0008.  The diff thinks a dummyapp migration was moved.

Reply:
thanks for the comment i was very confused